commands/daemon.go

```diff
@@ -133,7 +149,8 @@ func runAPIAndWait(ctx context.Context, node *node.Node, config *config.Config,
 		// TODO: should this be the passed in context?
 		ctx:          context.Background(),
 		api:          api,
-		porcelainAPI: node.PorcelainAPI,
+		porcelainAPI: nd.PorcelainAPI,
+		protocolAPI:  &protocolAPI,
```

Comment:
it might make sense to shoot for separate api containers for each protocol to make it easier to pull out or use individual protocols, eg if you wanted to embed just the retreival protocol in your binary. no action required i dont think.

Reply:
it's self-contained right now anyway so I could just rename it.